### PR TITLE
Editing toppings

### PR DESCRIPTION
**Pull request by lukasznowak85**
Possibility to edit toppings and toppings groups

### CODE REVIEW

File: src/components/ToppingsGroups.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <template>
     <div>
-        <h3>Składniki</h3>
-
+        <h3>Składniki <mdl-button iconn accent @click="toggleEditMode">EDYTUJ</mdl-button></h3>
```
Comment (piotros) [inline src/components/ToppingsGroups.vue]: `iconn`?

File: src/components/ToppingsGroups.vue [hunk 2]
```diff
@@ -20,10 +20,19 @@
...
+                editMode: false
+            }
+        },
         props: [ 'toppingsData', 'remove'],
         methods: {
             removeToppingsGroup(item) {
                 this.toppingsData.$remove(item)
+            },
+            toggleEditMode() {
+                console.log('ejze', this.editMode)
```
Comment (piotros) [inline src/components/ToppingsGroups.vue]: Remove `console.log`.

File: src/components/Topping.vue [hunk 3]
```diff
@@ -1,11 +1,14 @@
 <template>
 	<div>
-	    {{topping.name}} <mdl-button icon @click="remove(this.topping)">&times;</mdl-button>
+		<span v-if="editMode">
+			<mdl-textfield :value.sync="topping.name"></mdl-textfield> 
+			<mdl-button icon @click="remove(this.topping)">&times;</mdl-button></span>
```
Comment (piotros) [inline src/components/Topping.vue]: Use proper formatting.

Comment (lukasznowak85) [inline src/components/Topping.vue]: done

File: src/components/ToppingsGroups.vue (hunk 2 above)
Comment (lukasznowak85) [inline src/components/ToppingsGroups.vue]: done, ejze

File: src/components/ToppingsGroups.vue (hunk 1 above)
Comment (lukasznowak85) [inline src/components/ToppingsGroups.vue]: donne

**Pull request merged**